src/Makefile.am

```diff
@@ -380,6 +381,7 @@ libbitcoin_node_a_SOURCES = \
   node/context.cpp \
   node/eviction.cpp \
   node/interfaces.cpp \
+  node/mempool_persist_args.cpp \
```

Comment:
04f72e90204c049d5d320388fda1f01e53417938: Add to iwyu CI?

Reply:
I didn't do this most because IWYU insists that:

```
node/mempool_persist_args.cpp should add these lines:
#include <memory>         // for allocator
```

And I'm not sure why or if we need it... Does @ryanofsky know?

Reply:
Ok nvm then. Looks like an iwyu bug